server/lib/payments.js

```diff
@@ -280,7 +280,7 @@ export async function createRefundTransaction(transaction, refundedPaymentProces
       },
     });
 
-    // Old tips did not have a "debt" transaction associated
+    // Tips directly collected (and legacy ones) do not have a "debt" transaction associated
```

Comment:
So what's happening to them?

I was looking at that comment this morning and it feels like it's trying to communicate something but is failing short.

Reply:
The goal of this comment is to explain the reason for this `if`, why do some platform tips don't have a debt? It seems explicit to me that we're not doing anything more if there's no debt associated (since we're not entering the `if`), but I can eventually add more text if it's not